### PR TITLE
Added some more verbosity to the examples

### PR DESCRIPTION
**Pull request by goern**
I added a little more verbosity to the example/template to express a few ideas I had on my mind. Will catchup on freenode.net #nulecule later on...

I added a dockerfile and annoteted it a little, moved all to artifacts and provider specific dirs.

Additionally I am using goern/atomicapp-centos7 as a base image for the example. It is build from github.com/projectatomic/atomicapp

I added some Labels to demonstrate their use to the Atomicapp

### CODE REVIEW

File: spec/0.0.2/examples/template/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM goern/atomicapp-centos7
```
Comment (aweiteka) [inline spec/0.0.2/examples/template/Dockerfile:1]: A real example is `FROM projectatomic/atomicapp`

**Comment by aweiteka** [comment]
LGTM

File: spec/0.0.2/examples/template/Dockerfile (hunk 1 above)
Comment (goern) [inline spec/0.0.2/examples/template/Dockerfile:1]: well, I just see vpavlin's and mine... no projectatomic content on hub.docker.io :/

Comment (goern) [inline spec/0.0.2/examples/template/Dockerfile:1]: I think we should follow OpenShift's lead and put the base image name in the atomicapp name... atomicapp-rhel7 atomicapp-centos7 ...

Comment (vpavlin) [inline spec/0.0.2/examples/template/Dockerfile:1]: Good point, we should..

**Comment by aweiteka** [comment]
LABEL repo for reference: https://github.com/projectatomic/ContainerApplicationGenericLabels

**Comment by vpavlin** [comment]
LGTM

**Pull request merged**